docs/source/reference/data-access/python-api-owslib.ipynb

```diff
@@ -69,11 +69,11 @@
     "for dataset in collections['collections']:\n",
     "    print(dataset['title'])\n",
     "\n",
-    "malawi_obs = oafeat.collection_items('mw-mw_met_centre.data.core.weather.surface-based-observations.synop')\n",
+    "malawi_obs = oafeat.collection_items('mw-mw_met_centre-test.data.core.weather.surface-based-observations.synop')\n",
```

Comment:
Not directly related to this PR but please **note that all references based on topic-hierarchy-dotpath are incorrect**, since the index of the collection is using dataset-id (metadata_id).

Reply:
Good catch.  I did a sweep across docs, pushed to this PR just now.